lib/utilities/constants.dart

```diff
@@ -8,4 +8,5 @@ const pokedexTitle = 'Pokedex';
 const imageURL = 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/home/[index].png';
 const indexKey = '[index]';
 
-const errorMessageText = 'Something went wrong';
+const snackBarErrorMessage = 'Something went wrong!';
+const pokemonOverviewPageErrorMessage = 'No Pokemons Available';

```

Comment:
this isn't an error message right, rather a label? kindly rename this to `noPokemonsAvailableLabel` instead 🙂